user/lookup_unix.go

```diff
@@ -0,0 +1,286 @@
+// Copyright 2011 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build darwin dragonfly freebsd !android,linux netbsd openbsd solaris
+// +build cgo
+
+package user
```

Comment:
Why can't yuou use the functions in the os and os/user package for this ?

Is it possible to back this package out and propose it as a followup please.